.github/workflows/main.yml

```
@@ -0,0 +1,72 @@
+---
+name: main
+
+defaults:
+  run:
+    # In addition to GitHub's default, also abort when using undefined variables, and output individual commands.
+    # For the default, see
+    # <https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell>.
+    shell: 'bash --noprofile --norc -euo pipefail -x {0}'
+
+'on':
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+
+  pull_request:
+    branches:
+      - '**'
+
+  pull_request_target:
+    branches:
+      - '**'
+
+jobs:
+  lint-and-test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version:
+          - php-version: 5.6 
+          - php-version: 7.0 
+          - php-version: 7.1 
+          - php-version: 7.2 
+          - php-version: 7.3 
+          - php-version: 7.4 
+          - php-version: 8.0
+          - php-version: 8.1
+
+    name: >-
+      Test
+      with PHP ${{ matrix.php-version.php-version }}
```

Do we really need line folding here?
```suggestion
    name: Test with PHP ${{ matrix.php-version.php-version }}
```